### ISSUE TITLE
Markdown Support For Annotations [FEAT]

### ISSUE THREAD

**Issue by freddyyeddy** [Kruptein/PlanarAlly#320]
**Is your feature request related to a problem? Please describe.**
When Building a Campaign I rely heavily on Annotations for DM Info to reduce how often i need to saw windows. The Annotation system works ok for this but its really frustrating due to its center alignment. I can work with it but I like to do sub lists/bullet points for DC saves or treasure in the room.  

**Describe the solution you'd like**
I'm ok with using the tokens with annotations for quick info. So I propose keeping the annotation system built into the tokens but using markdown language for displaying it. this will allow a clean display of information for players and DM's. Also allow us to change (per annotation/token) alignment of text.
TLDR:
     Use markup language to allow lists/links to be used in PA
     At the least let me choose alignment for Annotations

**Describe alternatives you've considered**
Ive been doing manual text padding with - to force the center aligned text to look more list like
EG: 
This is a List of Chest Loot---------
-----Pendants----------------------
-----Bag of Holding----------------

needless to say this is quite clunky.
**Additional context**
While markup support for annotations is really useful and needed it may just be a crutch. This tool may need just a separate annotation tool/system to be a true fix. I would just be happy with fixing the alignment issues and adding markup language support

**Comment by Kruptein** [comment]
Annotations can use some love and this is one of the things that could indeed improve it.

**Comment by adunis** [comment]
I would take the opportunity to also change the annotations UI. Here's a suggestion that matches the style of the whole interface. 

![Uploading pzS3gPb.png…]()

**Comment by Kruptein** [comment]
Might consider that yes :)

**Comment by 10leej** [comment]
Just some thoughts on this since it was added to the 0.23 milestone, if you do implement markdown make sure there's a text field saying that they should use markdown with a link to say something like this site  
https://www.markdownguide.org/  

I know markdown is fairly common, but even long time reddit users don't know the markdown syntax
